.cirrus.yml

```diff
@@ -150,15 +150,15 @@ vendor_task:
 
     # Runs within Cirrus's "community cluster"
     container:
-        image: "quay.io/libpod/gate:latest"
+        image: docker.io/library/golang:1.13
```

Comment:
we need golang:1.12.

I've a similar patch here:
https://github.com/containers/libpod/pull/4451

Reply:
Why do you need golang 1.12?

Reply:
We need golang `1.13`. That's the latest version of go and the one we should be using since more and more contributors are migrating to F31 etc.

Reply:
Ok lets get Giuseppe merged with 1.12 and then I will update this PR to move us to 1.13.

Reply:
In #4451 is suggesting using `1.12` because the CI is not yet using `1.13` for vendoring.

Reply:
Yes

Reply:
This does work on Buildah though?  Do we have newer images on Buildah then on Podman.  @vrothberg are you saying I need to do more then just run the make vendor in a golang:1.13 container?

Reply:
This change here is good as we're changing it to 1.13 and all future vendor updates will need that as well.

The other comment referred to a previous version of another PR which pinned it to 1.12.